### ISSUE TITLE
Improving the curvature baker

### ISSUE THREAD

**Issue by SIsilicon** [Jummit/curvature-baker#1]
Hey pal! Like you I've been looking all over the internet for an algorithm that generates a curvature map. I believe we've even been through the same stack overflow pages.
I can't work on this now, but I hope this helps you.
https://github.com/blender/blender/blob/594f47ecd2d5367ca936cf6fc6ec8168c2b360d0/intern/cycles/blender/blender_mesh.cpp#L541
Blender uses this function to generate a pointiness attribute data. Maybe you can reverse engineer to make a higher quality curvature baker. :)

**Comment by Jummit** [comment]
Wow, thanks! I considered looking for that, but I have no experience with Blenders source code. Good find!

Funnily enough, this algorithm is pretty close to what I'm currently doing, it just samples the curvature differently.

**Comment by SIsilicon** [comment]
Yeah. It also treats the pointiness as per vertex data, so interpolates cleanly along the mesh.
